src/appmixer/utils/ai/CreateSpeech/CreateSpeech.js

```
@@ -0,0 +1,33 @@
+'use strict';
+
+module.exports = {
+
+    receive: async function(context) {
+
+        const { input, voice, responseFormat, speed } = context.messages.in.content;
+        const apiKey = context.config.apiKey;
+
+        if (!apiKey) {
+            return new context.CancelError('Missing \'apiKey\' system setting of the appmixer.utils.ai module pointing to OpenAI. Please provide it in the Connector Configuration section of the Appmixer Backoffice.');
```

@DavidDurman there must be `throw` instead of the `return`